src/UI/Component/Input/Field/Factory.php

```diff
@@ -348,5 +348,64 @@ public function password($label, $byline = null);
 	 * @return \ILIAS\UI\Component\Input\Field\Select
 	 */
 	public function select($label, array $options, $byline = null);
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     A Date Input is used to enter dates.
+	 *   composition: >
+	 *     Date Input will render a text field with the placeholder-attribute
+	 *     indicating the specified format.
+	 *     Next to the text field, a Calendar Glyph will trigger a popover containing
+	 *     a graphical selector/date-picker.
+	 *     Depending on the given format, next to the date-picker a time-picker will be shown.
+	 *     If desired, the Date Input can be rendered with the Time Glyph instead of the Calendar Glyph.
+	 *   effect: >
+	 *     When clicking the glyph, a popover is shown with the days of the month.
+	 *     Within the popover, the user may navigate to prior and following months.
+	 *     When format contains time-values, the picker is rendered with a time-selection.
+	 *   rivals:
+	 *     Text field: Text Felds MUST NOT be used to input date-strings.
+	 *
+	 * context:
+	 *   Date Input is used in forms.
+	 *
+	 * rules:
+	 *   usage:
+	 *     1: When used as a time-only input, the glyph MUST be Time Glyph.
+	 *
+	 * ---
+	 * @param string 	$label   defines the label.
+	 * @param string 	$byline
+	 *
+	 * @return \ILIAS\UI\Component\Input\Field\Date
+	 */
+	public function date($label, $byline = null);
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     A Duration Input is used to enter a time span instead of a single point in time.
+	 *   composition: >
+	 *     A Duration Input is composed as a group of two Date Inputs.
+	 *   effect: >
+	 *     Dependant on the format, the inputs will accept dates, times or datetimes.
```

Comment:
There is something missing like:

The start point must take place before the endpoint. Invalid input will be corrected automatically.